sklearn/calibration.py

```diff
@@ -213,13 +219,20 @@ def fit(self, X, y, sample_weight=None):
         sample_weight : array-like of shape (n_samples,), default=None
             Sample weights. If None, then samples are equally weighted.
 
+        **fit_params : dict of string -> object
+            Parameters passed to the fit method of the underlying
+            classifier.
+
         Returns
         -------
         self : object
             Returns an instance of self.
         """
         X, y = indexable(X, y)
 
+        for key, val in fit_params.items():
```

Comment:
We can delay this part until and use `sklearn.utils.validation._check_fit_params` and select index at this moment.

Reply:
Not sure what you mean here.